gfs/benchmark/performance_test.go

```diff
@@ -0,0 +1,190 @@
+package benchmark
+
+import (
+	"bytes"
+	"gfs/chunkserver"
+	"gfs/chunkserver/services"
+	"gfs/client"
+	"gfs/master"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// ----------- BENCHMARK CONFIGURATIONS HERE --------------------------
+
+var masterServerPort = ":9000"
+var chunkServerPortBase = 10000
+var NUM_CHUNK_SERVERS = 3
+
+var NTRIALS = 1 // repeat test NTRIALS time to collect
```

Comment:
Increasing the number of trials (which loops the test case), causes a gRPC dialing error (can't dial to a certain address). It seems that it may be overwhelming gRPC.